packages/sdk/contractkit/src/kit.ts

```diff
@@ -273,6 +274,60 @@ export class ContractKit {
     return this.connection.signTypedData(signer, typedData)
   }
 
+  async getFeeCurrencyWhitelist(): Promise<StrongAddress[]> {
+    const [whitelist, celo] = await Promise.all([
+      this._web3Contracts.getContract(CeloContract.FeeCurrencyWhitelist),
+      this.contracts.getGoldToken(),
+    ])
+    const feeCurrencyWhitelist = (await whitelist.methods.getWhitelist().call()) as StrongAddress[]
+
+    // Making sure we always have at least CELO and a uniq list.
+    return [...new Set([celo.address, ...feeCurrencyWhitelist])].sort()
```

Comment:
hmm i dont think we are suppose to use CELO erc20 as a feeCurrency

Reply:
Yeah I'm not sure why I was thinking the whitelist shouldn't be empty, but I'll remove it.